os/include/tinyara/pm/pm.h

```diff
@@ -244,6 +244,10 @@
 /* Defines max length of device driver name for PM callback. */
 #define MAX_PM_CALLBACK_NAME    32
 
+#define PM_LOCK_PATH					"/proc/power/domains/0/pm_lock"
+#define PM_UNLOCK_PATH					"/proc/power/domains/0/pm_unlock"
+#define PM_SLEEP_PATH					"/proc/power/domains/0/enter_sleep"
```

Comment:
The DVFS file path is missing here, please add:
#ifdef CONFIG_PM_DVFS
#define PM_SLEEP_PATH					"/proc/power/domains/0/pm_tunefreq"
#endif

Reply:
Added, thankyou